guides/common/modules/con_provisioning-workflow.adoc

```diff
@@ -79,3 +79,10 @@ If you use image-based provisioning, the host does not follow the standard PXE b
 The compute resource creates a copy of the image for the host to use.
 Depending on image settings in {Project}, seed data can be passed in for initial configuration, for example using `cloud-init`.
 {Project} can connect using SSH to the host and execute a template to finish the customization.
++
+[NOTE]
+====
+When you delete a host via *Administer* -> *Settings* -> *Provisioning*, consider destroying the associated VM upon deletion as well.
+If you do not destroy the associated VM and attempt to create a new VM with the same resource name later, it fails, because the VM of that name still exists in {Project}.
```

Comment:
```suggestion
If you do not destroy the associated VM and attempt to create a new VM with the same resource name later, it will fail because the VM of that name still exists in the external compute resource. The existing VM could still be registered to {Project} using the standard Host registration workflow that you would use for any already provisioned Host.
```